pkg/client/redhatsso/client.go

```diff
@@ -0,0 +1,154 @@
+package redhatsso
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	serviceaccountsclient "github.com/redhat-developer/app-services-sdk-go/serviceaccounts/apiv1internal/client"
```

Comment:
@ziccardi : please can you check the issue with import the sdk

Reply:
@wtrocki any idea?